src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -1979,9 +1980,15 @@ internal void SetMaxLanguageVersion(ProjectId projectId, string? maxLanguageVers
             }
         }
 
-        internal void RefreshProjectExistsUIContextForLanguage(string language)
+        internal void RefreshProjectExistsUIContextForLanguage(string language, CancellationToken cancellationToken)
```

Comment:
Why not delete this in favor of just calling the async method from the other use?

Reply:
done.